### ISSUE TITLE
OIDC UI: fix accessibility violation

### ISSUE THREAD

**Issue by lavenal** [OpenLiberty/open-liberty#8432]
Fix accessibility violations resulted by running batchscan on the three OIDC UI endpoints + the login page.
